pom.xml

```diff
@@ -130,6 +130,14 @@
         <version>${version.org.apache.cassandra}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.wildfly.bom</groupId>
+        <artifactId>jboss-javaee-7.0-wildfly</artifactId>
+        <version>${version.org.wildfly}</version>
```

Comment:
Should any part of commons run in a container distinct from WF with `${version.org.wildfly}`? Otherwise, I would not want to manage javax.ejb.* separately.